app/src/main/java/org/transdroid/search/RssFeedSearch/ExtraTorrentAdapter.java

```diff
@@ -121,14 +121,12 @@ public String getSiteName() {
 		return "ExtraTorrent";
 	}
 
-	@Override
-	public boolean isPrivateSite() {
-		return false;
+	public AuthType getAuthType() {
+		return AuthType.NONE;
 	}
 
-	@Override
-	public boolean usesToken() {
-		return false;
+	public String[] getRequiredCookies() {
```

Comment:
Since many adapter are implementations of `RssFeedSearchAdapter` perhaps override the `getRequiredCookies` method there already (as default implementation that returns `null`)?

Reply:
Yeah, I was going to do that but forgot. I also considered using the new [Java 8 default method](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html). Would that be OK? What JVM level do you want to be compatible with?

Reply:
Looks like it's not trivial. Default methods are supposed to be supported fow all SDK levels but require the newest Android Toolchain. I implemented in RssFeedSearchAdapter for now

Reply:
OK, I figured it out. It requires some changes to gradle files. I think it's better if I send it a seperate PR after this one is merged.